File: UL/CppApp/System/Exception.cpp
#include "stdafx.h"
#include "System\Exception.h"
#include "System\Object.h"
#include "System\String.h"
Ref<System::String> System::Exception::get_Message()
{
	return _msg;
}
System::Exception::Exception(Ref<System::String>  msg)
{
	_msg = msg;
}
System::Exception::Exception()
{
	_msg = Ref<System::String>(new System::String(_T("")));
}

File: UL/Debug/Lua/System/Exception.lua
require "System"
System.Exception = class('System.Exception',System.Object)
function System.Exception:get_Message()
	do
		return _msg;
	end
end
Exception(msg)
	do
		_msg = msg;
	end
end
Exception()
	do
		_msg = System.String.new("");
	end
end
